swap_meet/clothing.py

```diff
@@ -1,2 +1,26 @@
-class Clothing:
-    pass
+from swap_meet.item import Item
+
+class Clothing(Item):
+
+    def __init__ (self, condition = 0 ):
+        super().__init__(condition = 0, category = "")
+        self.category = 'Clothing'
```

Comment:
Creating the `category` attribute here isn't necessary as we can have the parent constructor to create a `category` attribute for the Clothing instance by passing that value in the `super()` constructor. 

```suggestion
        super().__init__(condition = 0, category = "Clothing")
```